fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -233,69 +237,60 @@ protected RdfStream getResourceTriples(final int limit) {
 
         final LdpPreferTag ldpPreferences = new LdpPreferTag(returnPreference);
 
-        final RdfStream rdfStream = new RdfStream();
-
         final Predicate<Triple> tripleFilter = ldpPreferences.prefersServerManaged() ? x -> true :
             IS_MANAGED_TRIPLE.negate();
 
-        if (ldpPreferences.prefersServerManaged()) {
-            rdfStream.concat(getTriples(LdpRdfContext.class));
-        }
-
-        rdfStream.concat(filter(getTriples(TypeRdfContext.class), tripleFilter::test));
+        final List<Stream<Triple>> streams = new ArrayList<>();
 
-        rdfStream.concat(filter(getTriples(PropertiesRdfContext.class), tripleFilter::test));
+        streams.add(getTriples(PROPERTIES).filter(tripleFilter));
 
-        if (!returnPreference.getValue().equals("minimal")) {
+        if (returnPreference.getValue().equals("minimal")) {
+            if (ldpPreferences.prefersServerManaged()) {
+                streams.add(getTriples(SERVER_MANAGED)
```

Comment:
I must have missed the request "to align the enum to the expected Prefer contexts". I would suggest that the current approach does not exactly hit the mark because we see here that filtering and tweaking is still required to get the desired triples in the various Prefer contexts.
If we can come up with a clean and transparent approach along the lines of what you have started here, that is great. If not, I have no problem with mapping the `RdfContext` enum to the `*RdfContext` values.